frontend/WikiContrib-Frontend/src/components/nav.js

```diff
@@ -42,3 +42,15 @@ export const NavBar = () => (
     </Menu.Menu>
   </Menu>
 );
+
+
+
+export const Footer = () => (
+  <footer className="footer">
+     Built with <Icon className="heart" name="heart" /> by
+     <a className="contributor" href="https://phabricator.wikimedia.org/p/Rammanojpotla/" target="_blank" rel="noopener noreferrer">Rammanojpotla,</a>
```

Comment:
The spacing between names don;t look normal. Could we copy what pageviews tool folks are doing? There should just be one space after `,`